src/mui-composed/components/chip/ChipWithRJSFInput.tsx

```diff
@@ -0,0 +1,83 @@
+import { ClickAwayListener, Paper, Popper, SxProps } from "@mui/material";
+import { IChangeEvent } from "@rjsf/core";
+import { Form } from "@rjsf/mui";
+import { ErrorTransformer, RJSFSchema, ValidatorType } from "@rjsf/utils";
+import React, { useRef, useState } from "react";
+
+import ChipWithAction from "./ChipWithAction";
+
+type Item = {
+    name?: string;
+};
+
+interface FormPropsType {
+    jsonSchema: RJSFSchema;
+    uiSchema?: any;
```

Comment:
Try to import UiSchema type from @rjsf

Reply:
this type does not exist, will investigate using JSONSchema